### PR TITLE
[DROOLS-5443] Move DMN-PMML tests to a separate module

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

File: kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.spec.javax.xml.bind</groupId>
+      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <dependencyManagement>
```
Comment (gitgabrio) [inline kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml:82]: Hi @jiripetrlik 
Is this dependency management needed ?

**Review by gitgabrio — COMMENTED** [review]
@jiripetrlik 
Thanks, nice idea! Just one comment about the dependency management

File: kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml (hunk 1 above)
Comment (jiripetrlik) [reply kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml:82]: @gitgabrio 
Unfortunately yes, without it I'm getting the following error:
The project org.kie:kie-dmn-pmml-tests-trusty:7.40.0-SNAPSHOT (/home/jpetrlik/devel/github/kiegroup/drools/kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.kie:kie-pmml-dependencies:pom is missing. @ line 32, column 17

Maybe we can add this artifact to some higher level bom. But it is probably out of scope of this PR.

**Review by gitgabrio — APPROVED** [review]
Good for me, thanks!

**Comment by jiripetrlik** [comment]
@danielezonca @gitgabrio 
Thank you for reviews! Can we merge now?

File: kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml (hunk 1 above)
Comment (gitgabrio) [reply kie-dmn/kie-dmn-pmml-tests-parent/kie-dmn-pmml-tests-trusty/pom.xml:82]: @jiripetrlik 
see https://issues.redhat.com/browse/DROOLS-5464

**Pull request merged**